storage/client/queue/package-info.java

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+/**
+ * This package contains the classes to perform actions on Azure Storage Queue.
+ */
+package com.azure.storage.queue;
```

Comment:
This package-info.java file is in the entirely wrong directory. Again, generate JavaDoc and you would see that there are missing descriptions for packages in your project. This should also fail CheckStyle rules, so please be sure you've run CheckStyle, SpotBugs, and JavaDoc.

Reply:
Done with move package-info to right place.